chart/validator-plugin-azure/crds/validation.spectrocloud.labs_azurevalidators.yaml

```diff
@@ -3,7 +3,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.12.0
```

Comment:
I think I forgot to update the chart the past few PRs that were merged into `main`, so it's a good thing we're doing this now if we plan to use these new VM image features!

Reply:
It should be getting caught now by the Ensure Reviewable action